zebrad/src/components/mempool/storage/tests.rs

```diff
@@ -49,77 +50,75 @@ fn mempool_storage_basic_for_network(network: Network) -> Result<()> {
     let mut storage: Storage = Default::default();
 
     // Get transactions from the first 10 blocks of the Zcash blockchain
-    let (total_transactions, unmined_transactions) = unmined_transactions_in_blocks(10, network);
+    let unmined_transactions: Vec<_> = unmined_transactions_in_blocks(..=10, network).collect();
+    let total_transactions = unmined_transactions.len();
 
     // Insert them all to the storage
     for unmined_transaction in unmined_transactions.clone() {
         storage.insert(unmined_transaction)?;
     }
 
+    // Separate transactions into the ones expected to be in the mempool and those expected to be
+    // rejected.
+    let rejected_transaction_count = total_transactions - MEMPOOL_SIZE;
+    let expected_to_be_rejected = &unmined_transactions[..rejected_transaction_count];
```

Comment:
This will break when we implement ZIP-401

Reply:
In this case it might make sense to change the assertions to be:

1. Does the storage contain exactly MEMPOOL_SIZE transactions?
2. Are all transactions in the mempool storage from the `unmined_transactions` list?
3. (Maybe) Are they all unique?